maintenance/projects/js-toolkit/packages/liferay-npm-build-tools-common/src/project/__tests__/index.test.js

```diff
@@ -10,9 +10,9 @@ import path from 'path';
 import PkgDesc from '../../pkg-desc';
 import {Project} from '../index';
 
-////////////////////////////////////////////////////////////////////////////////
-// Tests by property
-////////////////////////////////////////////////////////////////////////////////
+/**
+* Tests by property
```

Comment:
This is missing a space to the left. Maybe that's the cause for the SF failure...

Reply:
🤦